src/anomalib/deploy/inferencers/openvino_inferencer.py

```diff
@@ -247,8 +247,10 @@ def post_process(self, predictions: np.ndarray, metadata: dict | DictConfig | No
         # Common practice in anomaly detection is to assign anomalous
         # label to the prediction if the prediction score is greater
         # than the image threshold.
+        pred_label: str | None = None
```

Comment:
and looks like this is already defined in line 234. but the annotation should be 

```python
pred_label: LabelName | None = None
```

Reply:
@samet-akcay Not sure how to proceed here, how do I propose the updated annotation of pred_label as a code change?

Reply:
I created a pull request on your own fork. If you merge it, this PR will be automatically updated.

Can you merge this one on your fork?
https://github.com/TurboJonte/anomalib/pull/1